lib/deqm_test_kit/bulk_import.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+require_relative '../utils/bulk_import_utils'
+module DEQMTestKit
+  # BulkImport test group ensure the fhir server can accept bulk data import requests
+  class BulkImport < Inferno::TestGroup
+    id 'bulk_import'
+    title 'Bulk Import'
+    description 'Ensure the fhir server can accept bulk data import requests'
+
+    input :measure_id
+    custom_headers = { 'X-Provenance': '{"resourceType": "Provenance"}', prefer: 'respond-async' }
+    params = {
+      resourceType: 'Parameters',
+      parameter: [
+        {
+          name: 'measureReport',
+          resource: {
+            resourceType: 'MeasureReport',
+            measure: 'http://hl7.org/fhir/us/cqfmeasures/Measure/EXM130'
+          }
+        },
+        {
+          name: 'exportURL',
+          valueString: 'https://bulk-data.smarthealthit.org/eyJlcnIiOiIiLCJwYWdlIjoxMDAwMCwiZHVyIjoxMCwidGx0IjoxNSwibSI6MSwic3R1IjozLCJkZWwiOjB9/fhir'
+        }
+      ]
+    }
+    fhir_client do
+      url :url
+      headers custom_headers
+    end
+    # rubocop:disable Metrics:BlockLength
+    test do
+      title 'Ensure data can be accepted'
+      id 'bulk-import-01'
+      description 'Send the data to the server and the response is a 202 (or is it?)'
+      run do
+        assert(measure_id,
+               'No measure selected. Run Measure Availability prior to running the bulk import test group.')
+        fhir_read(:measure, measure_id)
+        assert_valid_json(response[:body])
```

Comment:
Are we checking that the measure exists on the server here? If so, I would probably assert a 200 response.

Reply:
Yes, I can add the assert 200, I had just used the same logic the submit data suite was using.